cmd/goal/node.go

```diff
@@ -491,11 +492,14 @@ func makeStatusString(stat model.NodeStatusResponse) string {
 			if stat.UpgradeYesVotes != nil {
 				upgradeYesVotes = *stat.UpgradeYesVotes
 			}
+			if stat.UpgradeVoteRounds != nil {
+				upgradeVoteRounds = *stat.UpgradeVoteRounds
+			}
 			statusString = statusString + "\n" + fmt.Sprintf(
 				infoNodeStatusConsensusUpgradeVoting,
 				upgradeYesVotes,
 				upgradeNoVotes,
-				upgradeNextProtocolVoteBefore-stat.LastRound,
+				upgradeVoteRounds-upgradeYesVotes-upgradeNoVotes,
```

Comment:
yes.

Reply:
upgradeVoteRounds was calculated in algod so this more direct formula can be used in the client